src/mongo/db/ldap_options.idl

```diff
@@ -136,3 +140,8 @@ configs:
         short_name: ldapReferrals
         arg_vartype: Bool
         default: false
+    'security.ldap.max_pool_size':
```

Comment:
MongoDB does not use this style of names for its parameters. See examples here: https://docs.mongodb.com/manual/reference/parameters/#param.ldapUseConnectionPool
Command line parameter should be something like "ldapConnectionPoolSize" and config file name should be something like "security.ldap.connectionPool.size"
BTW we have ldapUseConnectionPool parameter which is a No-op now.

Reply:
Do we really have platforms where we don'nt have `libldap_r`? Because then we also have issues with the curret non pooled implementation, as part of the authentication code will run on multiple threads.